.travis.yml

```diff
@@ -17,3 +18,4 @@ script:
   - npm test
   - cd ..
   - bash ./deploy.sh
+  - curl -O https://sideshowbarker.net/nightlies/jar/vnu.jar && /usr/lib/jvm/java-8-oracle/jre/bin/java -jar vnu.jar --skip-non-html /home/travis/build/whatwg/streams/streams.spec.whatwg.org
```

Comment:
If we use the command-line checker (jar) I don’t know of any way in Travis to avoid the need for Travis to download the jar each time.

If we wanted to avoid the download, we could switch (back) to using the HTTP API to https://checker.html5.org/, in which case the documents are POSTed to there for checking.

The reason I switched away from doing is that way is, we lose the path information for the files, and only get the file names reported. In some cases that means we can’t determine which file actually had errors (without actually examining the files).